specification/dnsresolver/resource-manager/Microsoft.Network/preview/2020-04-01-preview/dnsresolver.json

```diff
@@ -2383,7 +2383,6 @@
           "description": "Private IP address allocation method.",
           "type": "string",
           "enum": [
-            "Static",
```

Comment:
We only want to remove this from the examples in the documentation, not the API swagger.